station/app/api/api_v1/endpoints/airflow.py

```diff
@@ -77,4 +90,6 @@ def get_airflow_task_log(dag_id: str, run_id: str, task_id: str):
     @return:
     """
     run_info_data = airflow_client.get_task_log(dag_id, run_id, task_id)
+    if not run_info_data:
+        raise HTTPException(status_code=404, detail=f"{task_id} not known.")
```

Comment:
not found

Reply:
404 is not found. Or should this be changed?

Reply:
Not found in the message 😄